src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -231,11 +231,11 @@ protected virtual Task<bool> IsSemanticTriggerCharacterAsync(Document document,
             return SpecializedTasks.True;
         }
 
-        private Task<ImmutableArray<ISymbol>> GetSymbolsWorker(int position, bool preselect, SyntaxContext context, OptionSet options, CancellationToken cancellationToken)
+        private async Task<ImmutableArray<(ISymbol, CompletionItemRules)>> GetSymbolsWorker(int position, bool preselect, SyntaxContext context, OptionSet options, CancellationToken cancellationToken)
         {
             return preselect
-                ? GetPreselectedSymbolsWorker(context, position, options, cancellationToken)
-                : GetSymbolsWorker(context, position, options, cancellationToken);
+                ? await GetPreselectedSymbolsWorker(context, position, options, cancellationToken).ConfigureAwait(false)
+                : (await GetSymbolsWorker(context, position, options, cancellationToken).ConfigureAwait(false)).Select(s => (s, CompletionItemRules.Default)).ToImmutableArray();
```

Comment:
lines not long enugh to fit in github should be wrapped :)

Also, i personally fine 'await' as a nested expression pretty hard to read/reason about.  

When code gets this complex, just convert to statement form.  THanks!